kuma/payments/jinja2/payments/thank_you.html

```diff
@@ -71,8 +74,61 @@ <h2 class="highlight highlight-spanned">
 
 {% block content %}
     <main id="contributions-page" class="contributions-page">
+        <!-- TODO: replace with waffle flag/varible -->
+        {% if False %}
+            <section class="section">
+                <div class="index">
+                    <h2>{{_('Useful Things')}}</h2>
+                </div>
+                <div class="faqs clear feedback">
+                    <div class="faq-feedback">
+                        <div class="sub-section">
+                            <h3>{{_('Managing your payments')}}</h3>
+                            <p>
+                                {% trans url="mailto:mdn-support@mozilla.com?Subject=Manage%20monthly%20payment" %}
+                                    If you would like to manage your monthly payment,
+                                    such as changing your card account details or the
+                                    amount you pay, please contact
+                                    <a href="{{ url }}">mdn-support@mozilla.com</a>.
```

Comment:
and then this changes to

```suggestion
                                    <a href="{{ url }}">{{ email }}</a>.
```